package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "packageManager": "pnpm@8.7.1",
+  "packageManager": "pnpm@9.12.3",
```

Comment:
Can you clarify why we need the pnpm version bump? Is mermaid incompatible with older versions?

Reply:
With `pnpm@8.7.1`, running docker-compose fails with the following error:
```bash
0.280 ! Corepack is about to download https://registry.npmjs.org/pnpm/-/pnpm-8.7.1.tgz
1.109  ERR_PNPM_LOCKFILE_BREAKING_CHANGE  Lockfile /ozone-site/pnpm-lock.yaml not compatible with current pnpm
1.109 
1.109 Run with the --force parameter to recreate the lockfile.
------
failed to solve: process "/bin/sh -c pnpm install --prod --frozen-lockfile" did not complete successfully: exit code: 1
`docker-compose` process finished with exit code 17
```

I added the docusaurus/theme-mermaid package via `pnpm add @docusaurus/theme-mermaid@3.3.2` on my mac. This updated the `pnpm-lock.yaml` and the `package.json` files.